.github/settings.yml

```diff
@@ -0,0 +1,62 @@
+repository:
+  name: configuration-as-code-plugin
+  description: Jenkins Configuration as Code Plugin
+  topics: 'configuration-as-code, jenkins, hacktoberfest, jenkins-configuration, jcasc'
+  private: false
+  default_branch: master
+  has_issues: true
+  has_wiki: false
+  allow_merge_commit: false
+  allow_squash_merge: true
+  allow_rebase_merge: false
+
+labels:
```

Comment:
Could we move it to https://github.com/jenkinsci/.github ? IIUC the app supports Probot, and having some default labels would not hurt if others from Web UI do not get removed

Reply:
https://github.com/probot/settings/issues/107
Currently, these cannot be moved to .github as they would delete other labels that are not inside settings.yml

Reply:
```suggestion
#TODO(oleg-nenashev): Consider moving to https://github.com/jenkinsci/.github once https://github.com/probot/settings/issues/107 is fixed
labels:
```